doc/releases/octopus.rst

```diff
@@ -124,9 +124,18 @@ RADOS
 **RGW** object storage
 ~~~~~~~~~~~~~~~~~~~~~~
   
-* Multi-site replication can now be managed on a per-bucket basis (EXPERIMENTAL).
-* WORM?
-* bucket tagging?
+* New `Multisite Sync Policy`_ primitives for per-bucket replication. (EXPERIMENTAL)
+* S3 feature support:
+    - Bucket Replication (EXPERIMENTAL)
+    - Bucket Tagging
+    - Object Lock
+    - Public Access Block for buckets
+* Bucket sharding:
+    - Significantly improved listing performance on buckets with many shards.
+    - Dynamic resharding prefers prime shard counts for improved distribution.
+    - Raised the default number of bucket shards to 11.
```

Comment:
maybe worth adding the content in PendingReleaseNotes about this as we've added in https://github.com/ceph/ceph/pull/32660 about the zone params

Reply:
they'll already see that stuff in release notes